src/Mutation.tsx

```diff
@@ -35,14 +35,19 @@ export declare type MutationUpdaterFn<
   }
 > = (proxy: DataProxy, mutationResult: FetchResult<T>) => void;
 
-export declare type FetchResult<C = Record<string, any>, E = Record<string, any>> = ExecutionResult<
-  C
-> & {
+export declare type FetchResult<
+  TData = Record<string, any>,
+  C = Record<string, any>,
+  E = Record<string, any>
+> = ExecutionResult<TData> & {
   extensions?: E;
   context?: C;
 };
 
-export declare type MutationOptions<TData = any, TVariables = OperationVariables> = {
+export declare type MutationOptions<
+  TData = { [key: string]: any },
```

Comment:
Any reason not to use `Record<string, any>` here (just for consistency)?

Reply:
No reason at all - I'll change it, thanks!